source/tutorials/tutorial.md

```diff
@@ -65,8 +67,16 @@ If you're here, **CONGRATULATIONS!** You have already <a href="#install">install
 
 <p style="text-align:center;"><img src="../../assets/images/tutorial/achievements.png" alt="Achievements"></p>
 
-So, the first thing you want to do is find the perfect spot to settle down. But, before you do that, there are two things you have to consider. The following comments are the most important you should read and consider to ensure a successful Town and avoid frustration. Please read all considerations first, before placing your [Supply Ship](../items/supplycampandship) or [Supply Camp](../items/supplycampandship). Even before starting to play:
+To avoid mistakes and frustation, read through this entire guide carefully before starting to play.
+
+
+### Placement of the Supply Ship or Supply Camp 
+
+To place the [Supply Ship](../items/supplycampandship), you need to have a large enough body of water (at least 32 x 20).
+To place the [Supply Camp](../items/supplycampandship), you need to have a large enough flat, clear piece of land (at least 16x17 blocks).
 
+When you place the Supply Ship/Camp, it will spawn a chest that will contain the [Town Hall](../../source/buildings/townhall) block and the [building tool](../../source/items/buildingtool). **Once the Supply Ship/Camp is placed, you can't place another in the world.**
```

Comment:
```suggestion
When you place the Supply Ship/Camp, it will spawn a chest that will contain the [Town Hall](../../source/buildings/townhall) block and the [building tool](../../source/items/buildingtool). **Once the Supply Ship/Camp is placed, you can't place another in this world (but other players can).**
```